src/reference-data/defaults.ts

```diff
@@ -17,3 +17,4 @@ export const defaultModesWeSellTicketsFor: string[] = [
   'shuttleBus',
 ];
 export const defaultPaymentTypes: string[] = ['vipps', 'visa', 'mastercard'];
+export const defaultVatPercent: number = 12;
```

Comment:
The vat percent, payment types and modes we sell tickets for are not actually reference data, and the defaults does not belong here. Rather move them closer to the `FirestoreConfigurationContext`, either in that file or in a `defaults.ts` in the same folder.

Reply:
Fixed, moved to a separated file in the same folder than `FirestoreConfigurationContext `